docs/components/huggingface-embeddings/deployment.md

```diff
@@ -0,0 +1,90 @@
+# Deploying and Using Huggingface embeddings
+
+To host huggingface embeddings we are using the
+[text-embeddings-inference](https://github.com/huggingface/text-embeddings-inference)
+by huggingface
+
+## How to deploy
+
+Before deploying huggingface embeddings, we need to make sure that the vector size generated by the new embeddings is the same as pgvector vector size in `PGVECTOR_VECTOR_SIZE`. If not, we can
+update the vector size by uncommenting it in `.core_backend.env` and updating it to the new
+value. Note that if the database is already existing, this will not work unless the database is
```

Comment:
```suggestion
value. Note that if the database is already set up using a different `PGVECTOR_VECTOR_SIZE` value, this will not work unless the database is
```